timescaledb/how-to-guides/ingest-data.md

```diff
@@ -1,100 +1,79 @@
-users the opportunity to try this functionality, we built
-[downloadable binaries](/timescaledb/latest/tutorials/telegraf-output-plugin/#setting-up-telegraf)
-of Telegraf with our plugin already included.
-</highlight>
-
-The PostgreSQL plugin extends the ease of use users get from leveraging Telegraf
-by handling schema generation and modification. This means that as metrics are
-collected by Telegraf, the plugin creates a table if it doesn't exist and alters
-the table if a schema has changed. By default, the plugin leverages a
-[wide model][wide-model], which is typically the schema model that TimescaleDB
-users tend to choose when storing metrics. However, you can specify that you
-want to store metrics in a narrow model with a separate metadata table and foreign keys. You can also choose to use JSONB.
-
-To get started with the PostgreSQL and TimescaleDB output plugin, visit the
-[tutorial][telegraf-tutorial].
-
-## PostgreSQL's Kafka connector
-Another popular method of ingesting data into TimescaleDB is through the use of
-the
-[PostgreSQL connector with Kafka Connect][postgresql-connector-kafka].
-The connector is designed to work with [Kafka Connect][kafka-connect] and to be
-deployed to a Kafka Connect runtime service. It's purpose is to ingest change
-events from PostgreSQL databases (i.e. TimescaleDB).
-
-The deployed connector monitors one or more schemas within a TimescaleDB
-server and write all change events to Kafka topics, which can be independently
-consumed by one or more clients. Kafka Connect can be distributed to provide
-fault tolerance to ensure the connectors are running and continually keeping
-up with changes in the database.
-
-<highlight type="tip">
-The PostgreSQL connector can also be used as a library without Kafka or
-Kafka Connect, enabling applications and services to directly connect to
-TimescaleDB and obtain the ordered change events. This approach requires the
-application to record the progress of the connector so that upon restart,
-the connect can continue where it left off. This approach may be useful for
-less critical use cases. However, for production use cases, it's recommended
-that you use this connector with Kafka and Kafka Connect.
-</highlight>
-
-To start using the PostgreSQL connector, visit the
-[GitHub page][github-debezium].
-If you are interested in an alternative method to ingest data from Kafka to
-TimescaleDB, you can download the
-[StreamSets Data Collector][streamsets-data-collector] and get started with
-this [tutorial][tutorial-streamsets].
+aggregate views, downsampling, data gap-filling, and interpolation.
+Additionally, Promscale supports Grafana using [Prometheus][prometheus-grafana]
+and [PostgreSQL][postgres-grafana] data sources.
+
+## Telegraf
+Telegraf collects, processes, aggregates, and writes metrics. Telegraf is highly
+extensible, and has over two hundred plugins for gathering and writing different
```

Comment:
What's our style guide? I'd lean towards numerals for "200" (marginally better readability, follows [Google's style guide](https://developers.google.com/style/numbers), which is what I think we lint against?) but not a strongly held opinion

Reply:
Let's roll with Google. I think I was harking back to Chicago on that one 😉